cmd/build.go

```diff
@@ -322,6 +323,12 @@ func GenerateKustomizeResult(config opConfig.Config, kustomizeTemplate template.
 			if strings.Contains(manifestFileContentStr, key) {
 				manifestFileContentStr = strings.Replace(manifestFileContentStr, oldRawString, rawStr, -1)
 			}
+
+			oldBase64String := "$base64(" + key + ")"
```

Comment:
Assuming this won't be needed if we can use `stringData`?

Reply:
That's a maybe. I ran into an issue with google's clouddns, it didn't like having the json string, complaining about formatting. Converting it into base64 solved the issue, but it's probably unnecessary. Need to test. Maybe it's as simple as making it a single line string.